HW17/src/ThreadSafeList.java

```diff
@@ -0,0 +1,25 @@
+import java.util.LinkedList;
+import java.util.List;
+
+public class ThreadSafeList<T> {
```

Comment:
The class must implement the List interface in a thread-safe manner. Methods add, get and remove must be implemented. The rest of the methods can be empty/not implemented (for example, throw OperationNotSupportedException).

Reply:
I see, thanks!